pkgs/top-level/python-packages.nix

```diff
@@ -8173,7 +8173,7 @@ in {
 
   google_gax = callPackage ../development/python-modules/google_gax { };
 
-  gpg = toPythonModule (pkgs.gpgme.override { withPython=true; });
+  gpgme = toPythonModule (pkgs.gpgme.override { withPython=true; });
```

Comment:
@teto do you agree with change?

Reply:
I hesitated between the 2 and ended up with `gpg` because it's `import gpg ` and it's like the official ports for gpg (even if gpgme expose a simpler interface). So I have no problem with changing the name :)